### ISSUE TITLE
Print a name tag and not a pickuptag

### ISSUE THREAD

**Issue by mbyrdLCS** [ChurchApps/ChurchAppsSupport#270]
When you edit a group you have the option to print the tags with the parent pickup we also need an option to print just name tags no parent pickup. Use case would be if you were using this to print a tag for every adult in the church they do not need pickup tags. 

<img width="1044" alt="Screenshot 2024-04-18 at 10 55 28 AM" src="https://github.com/ChurchApps/ChurchAppsSupport/assets/65249159/e0e8c119-f57b-4da1-8660-fd59d2a6c0fc">

**Comment by jzongker** [comment]
@mbyrdLCS rather than adding this to each group, I think we should have a single setting on the Service churches define indicating whether name tags should be printed for everyone.  Do you agree?

**Comment by mbyrdLCS** [comment]
Yes I think that will work. Default to just kids classes like it is.

**Comment by Himali-Malvawala** [comment]
I've added the field to the groups.

**Comment by jzongker** [comment]
This is available now in version 1.0.2 of B1Checkin
